README.md

```diff
@@ -297,6 +297,11 @@ Git repositories to clone to this docker image. By default, each repo will be
 cloned to the provided name under `/workspace`. However, you may also specify a
 new subdirectory to home it in.
 
+If your repository is private you may provide the credentials or ssh_key to gain
+access. You may also avoid including these private details in the yaml by
+passing them through host environment variable. Check the ```Yaml loader``` section for
```

Comment:
This should also mention that both methods avoid including this information inside the docker image once built.

Reply:
i mentioned that login details are removed